examples/vector_database/compute_vectors.yaml

```
@@ -0,0 +1,49 @@
+name: clip-batch-compute-vectors
+
+workdir: .
+
+resources:
+  accelerators: 
+    # ordered by pricing (cheapest to most expensive)
+    T4: 1
+    L4: 1
+    A10G: 1
+    A10: 1
+    V100: 1
+  memory: 32+
+  any_of:
+    - use_spot: true 
+    - use_spot: false
+
+num_nodes: 1
+
+file_mounts:
+  /output:
+    name: sky-demo-embedding
+    # this needs to be the same as the source in the build_vectordb.yaml
+    mode: MOUNT
+  /images:
+    name: sky-demo-image
+    # this needs to be the same as the source in the build_vectordb.yaml
+    mode: MOUNT
+
+envs:
+  # These env vars are required but should be passed in at launch time.
+  HF_TOKEN: ${HF_TOKEN}
+  START_IDX: ${START_IDX}
+  END_IDX: ${END_IDX}
```

```suggestion
  HF_TOKEN: ''
  START_IDX: ''
  END_IDX: ''
```
Maybe this will work instead